salt/orch/kubernetes.sls

```diff
@@ -131,6 +131,16 @@ reboot_setup:
     - require:
       - salt: kube_master_setup
 
+dex_setup:
```

Comment:
This block is unnecessary, dex is in the highstate via `salt/top.sls`, which is ran against the masters 3 states up.